src/test/java/com/coyoapp/tinytask/service/DefaultFileServiceTest.java

```diff
@@ -0,0 +1,54 @@
+package com.coyoapp.tinytask.service;
+
+import com.coyoapp.tinytask.exception.FileException;
+import com.coyoapp.tinytask.repository.FileRepository;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.rules.TemporaryFolder;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.junit.MockitoJUnit;
+import org.mockito.junit.MockitoRule;
+import org.mockito.quality.Strictness;
+
+import java.util.Optional;
+
```

Comment:
hamcrest assertThat and mockito matchers instead of AssertJ (see already existing tests)